src/halmos/cheatcodes.py

```diff
@@ -687,7 +690,7 @@ def handle(sevm, ex, arg: ByteVec, stack, step_id) -> ByteVec | None:
 
             return ret
 
-        # getBlockNumber() return (uint256)
+        # vm.getBlockNumber() return (uint256)
         elif funsig == hevm_cheat_code.get_block_nunber_sig:
```

Comment:
unrelated, but typo here

```suggestion
        elif funsig == hevm_cheat_code.get_block_number_sig:
```

Reply:
(needs to be fixed at definition site too)

Reply:
good catch, fixed, thanks!